src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: nowrap;
+  height: 60px;
+}
+
+.nav__list {
+  padding: 0;
+}
+
+img {
+  margin: 40px;
+}
+
+.nav {
+  margin: 0 50px;
+  /* padding: 0 10px; */
+}
+
+.nav__item {
+  display: inline-block;
+  position: relative;
+  padding-left: 20px;
+}
+
```

Comment:
after effect is for `nav__link` so use position ralative for that not `nav__item`

Reply:
done